src/client/src/ui/blotter/components/Blotter.tsx

```diff
@@ -83,11 +67,19 @@ const Blotter: React.FC<BlotterProps> = props => {
   const [displayedRows, setDisplayedRows] = useState(0)
   const [gridDoc] = useState(React.createRef<HTMLDivElement>())
   const [gridApi, setGridApi] = useState(null)
+  const platform = usePlatform()
   const onModelUpdated = useCallback(
     () => gridApi && setDisplayedRows(gridApi.getDisplayedRowCount()),
     [gridApi],
   )
 
+  const broadcastContext = (context: Context) => {
+    if (platform.name !== 'openfin') {
+      return
+    }
+    platform.fdc3.broadcast(context)
```

Comment:
can you change to 
const broadcastContext = (context: Context) => platform.broadcast(context)

then have the non openfin adapters do nothing on broadcast  so your not checking platform name in the component